section2/exercises/if_statements.rb

```diff
@@ -61,5 +63,11 @@
 # Experiment with manipulating the value held within both variables
 # to make sure all above conditions output what you expect.
 
-cups_of_flour = 1
-has_sauce = true
+cups_of_flour = 2
+has_sauce = false
+
+if cups_of_flour >= 2 && has_sauce == true
```

Comment:
Beautiful!

Your use of white space, indentation, hard returns - PERFECT Ruby conventions - makes it so readable!